main.css

```diff
@@ -348,3 +348,17 @@ body {
 	overflow-y: scroll;
 	background-color: #eee;
 }
+
+#palette-select {
```

Comment:
For the style, this is fine since consistent with mine, but I will most likely make it so that we do _not_ style based on id, but rather classname

I forget where I learned that but think it came from this amazing HTML & CSS course: https://www.internetingishard.com/html-and-css/

Reply:
That makes a lot of sense. It will also cut down on the amount of code being written since I repeated a couple stylings before. Does this look better?